django_forest/utils/apimap_errors.py

```diff
@@ -0,0 +1,14 @@
+APIMAP_ERRORS = {
+    0: {
```

Comment:
`0` normally means "no error". You're sure about this value?

Reply:
This is something I have seen in the `forest-express` liana: https://github.com/ForestAdmin/forest-express/blob/master/src/services/apimap-sender.js#L15

`forest-rails` does not manage it this way. I will review this part for making it more pythonic thanks to `requests`